app/src/main/kotlin/com/wire/android/di/AppModule.kt

```diff
@@ -82,4 +84,11 @@ object AppModule {
     @Singleton
     @Provides
     fun provideCurrentTimestampProvider(): CurrentTimestampProvider = { System.currentTimeMillis() }
+
+    @Provides
+    fun provideGeocoder(appContext: Context): Geocoder = Geocoder(appContext)
+
+    @Singleton
```

Comment:
does it need to be a Singleton? maybe using Reusable if ViewModelScope is not possible to be used here

Reply:
It's not required but these parameters shouldn't change anywhere in the app, it's just a data class with values, do you see any disadvantages of it being a singleton?

Reply:
the disadvantages will be the instance of that class will never be garbage collected once they are crewated